ci-operator/config/openstack-k8s-operators/designate-operator/openstack-k8s-operators-designate-operator-main.yaml

```diff
@@ -29,6 +66,17 @@ tests:
     pre-commit run -a --show-diff-on-failure
   container:
     from: openstack-operator-ci-pre-commit
+- as: designate-operator-build-deploy-kuttl
+  cluster_claim:
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.10"
```

Comment:
I think that the failure is not random and there are no more 4.10 sno clusters available? This should maybe be `4.12` as in https://github.com/openshift/release/blob/90a763ca619811e5f847db4b9c6010dc35706643/ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main.yaml#L83? @Sandeepyadav93 could you clarify?

Reply:
correct, we moved to 4.12

Reply:
@bshewale please update the version in line 76 as well

Reply:
Done